[우연] 2차 과제 제출

dev-wooyeon opened the pull request:
https://github.com/dev-wooyeon/youth-application/blob/mission2-fix/미션_2.md

리뷰 남겨 주시기 전에는 요기로 봐주시면 보기 편하실 것 같습니다.

woowahan-pjs committed:
Initial commit

dev-wooyeon committed:
[미션2차] 뱅크샐러드 작성

dev-wooyeon committed:
Merge branch 'woowahan-pjs:main' into mission2-fix

dev-wooyeon commented:
https://github.com/dev-wooyeon/youth-application/blob/1bee2f0392d66214fe4e78c239956db3ab310218/미션_2.md

요기로 봐주시면 보기 편하실 것 같습니다.

dev-wooyeon committed:
[미션2차] 에이블리 작성

dev-wooyeon committed:
Revise 미션_2.md with new formatting and images

Updated formatting and added images to the document.

dev-wooyeon committed:
[미션2차] 에이블리 작성

dev-wooyeon committed:
Merge remote-tracking branch 'origin/mission2-fix' into mission2-fix

dev-wooyeon committed:
이미지 크기 수정

dev-wooyeon committed:
Update 미션_2.md

devFancy reviewed (approved):
안녕하세요 우연님!
먼저 `뱅크샐러드`와 `에이블리`라는 회사에 대한 자료를 작성하시느라 고생 많으셨어요 !

전반적으로 글을 읽으면서 인상 깊었던 점은 아래와 같아요.
- 회사를 이해(1~4번 질문)해보는 것뿐만 아니라, 본인이 이 회사와 적합한지 테스트 및 정리한 부분
  - 예시. "나는 왜 뱅크샐러드에 가고 싶은가?" -> 프롬프트를 활용해서 적합도 판단한 부분
- 회사에서 나온 자료를 기반으로 정리한 것뿐만 아니라, 우연님의 시선과 생각으로 정리한 점이 좋았어요.


충분히 고민한 흔적들이 많아서 그런지, 내용적인 면에서 크게 보완할 부분은 없다고 생각해요.
- 지금 작성하신 내용을 다듬기만 하면 좋겠다는 생각이 들었어요.
- "각 서비스의 조직도를 직접 예측해 보세요." 부분 혹은 뒷부분에서 우연님이 지원하고자 하는 직무의 채용 공고를 분석해보면서, 구체적으로 어떠한 인재상을 원하는지 작성해보면 좋을 것 같아요.

덕분에, 뱅크샐러드와 에이블리의 회사에 대해 보다 자세히 알게 되어서 감사해요 ☺️

dev-wooyeon committed:
fix: 에이블리 이미지 오기입 되어있는 것 수정

youth-simon commented on 미션_2.md:
```diff
@@ -0,0 +1,292 @@
+- 금융 데이터를 폭 넓게 다루면서도 높은 기술력으로 추후 토스, 네이버 파이낸스, 카카오페이 등 커리어를 이어갈 방향이 많아보임. (금융 도메인을 좋아하는 사람의 경우겠지만요.)
+- 비교적 금융 도메인은 산업에서 높은 프리미엄을 가지는 것 같다.
+
+[적합도 판단(w/ AI).md](./resources/뱅크샐러드_적합도_분석.md)
```
적합도 판단은 중요한 것 같아요!

- 다만, jd나 유튜브, 텍스트에서 나타내지 못한 팀의 세부사항들을 어떻게 잡을지도 고민해보시면 좋을것같아요!
- 결국에는 커피챗과 같은 경로로, 실제로 필터링된 요구사항 보다 raw data 그대로의 요구사항을 들어볼 기회가 있으면 좋을것같아요!

youth-simon commented on 미션_2.md:
```diff
@@ -0,0 +1,292 @@
+대표이사(CEO)
+ ├─ 기술총괄(CTO)
+ │    ├─ 엔지니어링 부서
+ │    │    ├─ 백엔드 엔지니어링팀
```
'뱅크샐러드' 는 백엔드 엔지니어링팀 과 프로덕트 부서가 분리가 되어있는것일까요?

youth-simon commented on 미션_2.md:
```diff
@@ -0,0 +1,292 @@
+> 
+- “우리는 고객의 가장 큰 이익을 보증하는 혁신으로 위대한 변화를 만들어갑니다.”
+- 국내 최초로 만든 금융의 표준
+> 프로덕트 중심!
```
해당 목표를 위한 여러가지 디테일들이 있을것같아요

예를 들면, 장기 프로덕트 계획이라던지, 어떤 내부 문제들을 해결하고 있는지

이런 부분들을 찾을 수 있으면, 더 좋을것같아요!

youth-simon reviewed (commented):
안녕하세요 우연님!

정성깊은 과제 내용이 인상깊었어요!

1. 적합성 분석
2. 인재상 및 원칙
3. 사용자 목적별 서비스 사용


만약에, 시간이 조금 더 남으신다면, 기업의 장기적 과제나 인재상이 어떤 맥락에서 정해진것인지 
정도를 추가 해볼 수 있을것같아요


하지만, 그런 내용들은 실제로 조직 구성원가 이야기하지 않는이상 알아낼 수 없다고 생각해요
진심으로 관심있는 회사라면, 커피챗을 한번 해보는것도 추천드려요!

dev-wooyeon committed:
Update 미션_2.md

dev-wooyeon committed:
Update 문서_2

dev-wooyeon committed:
Update 미션_2.md

dev-wooyeon committed:
Update 미션_2.md

Updated job posting analysis for data engineer positions, including role overview, required qualifications, and ideal candidate profile.

dev-wooyeon committed:
Update 미션_2.md
